src/modules/commentsCounter.js

```diff
@@ -0,0 +1,4 @@
+export default (arr) => {
+  const commentNum = document.querySelector('.comment-number');
+  commentNum.textContent = arr.length;
+};
```

Comment:
- [ ] Great job working on the project so far but I have small note to add so please in the comment pop up I think you have the wireframe a little bit different than the given wireframe since you should just display the image and then 4 piece of information above the inputs to add a new comment since I can see the image and beside it a description and below the input but in the requirements you should follow the layout of the wireframes provided. so please fix it 🤩💪🏻✅

## The items should be vertically aligned like the given wireframe
- Your design
![image](https://github.com/forLoop94/Javascript_capstone/assets/58553711/8545df72-cdb6-4515-8ccb-13e70d950cb2)

- Given
![image](https://github.com/forLoop94/Javascript_capstone/assets/58553711/016e2697-de8d-43c4-95cd-8bc04ee13314)

Reply:
Hi @Reem-lab 
Thank you for reviewing our PR and highlighting where we can make changes so we were able to make all the changes requested
Cheers

Reply:
- [ ] Great job effecting the changes Team. I think we still have one issue to resolve before we can proceed. When you look at the comment pop up, there's a lot of space after the title of the movie (Between movie title and description) and also after the description and the heading Comments. Due to this, a user has to scroll the pop up alot to fill and submit their comment. This does not only make your design different from the given wireframe, it also does not give the very best user experience. Kindly fix this issue and we would be ready to move ahead.
![image](https://github.com/forLoop94/Javascript_capstone/assets/62849872/45762c8e-99c3-46eb-b2ba-41d0e2555a7e)